### DISCUSSION TITLE
rigctl error on Linux

### DISCUSSION THREAD

**Discussion by lpax** [AlexandreRouma/SDRPlusPlus#513] [Q&A]
Hi everyone,
I am trying this program which i really like as panadapter.
 I manage my RTXs (Yaesu FT450D and FT817ND) without problems with CQRlog, Fldigi, Jtdx and Gpredict through the combination of flrig and rigctld programs. Now when I activate the Rigctl Server function on sdrpp I get this message 
**[2021-11-13 08: 20: 50.330] [error] Could not start rigctl server: std :: exception**
The IP and port configuration is the standard one that I also use with other programs (**localhost** and **4532**).
The startup sequence of the programs is **flrig** -> **rigctld -m xx** -> **sdrpp** where xx is the value for the rtx 
The Linux distro is Mint 20.2 with XFCE.
Anyone use this feature on Linux? Am I wrong somewhere or is there a bug? 

Best regards, and thanks,
Luigi

**Comment by AlexandreRouma** [comment]
change the port to something else. Chances are if you're using rigctl for your rig the default port is already taken

**Reply by lpax** [reply]
Thanks for your answer, but I got confused.
I thought that the program could control another radio in order to use it as a panadapter, instead the Rigctl Server function it implemented serves to interface it like a normal radio to a program.

Is it possible to implement the Panadapter or Radio Control function using hamlib? 

Best regards, and thanks,
Luigi

**Reply by ON4BCB** [reply]
Hi Luigi

did you solved the problem already, would like to do the same

**Reply by the-gobleo-tinkerer** [reply]
Using SDR++ as a pandapter was discussed in #753.
